apps/grader-host/src/lib/config.js

```diff
@@ -52,6 +52,8 @@ const ConfigSchema = z.object({
   sentryDsn: z.string().nullable().default(null),
   sentryEnvironment: z.string().nullable().default(null),
   awsRegion: z.string().default('us-east-2'),
+  visibilityTimeout: z.number().default(60),
+  visibilityTimeoutHeartbeatIntervalSec: z.number().default(30),
```

Comment:
Would it be safe to set these to much shorter values to provide rapid failover in the case that something happens to our grader VM? Perhaps 10/5 seconds?

Reply:
My main concern here was rate limiting, which we've seen with other AWS APIs (namely S3). I checked the [SQS quotas](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-messages.html) and found the following:

> Standard queues support a nearly unlimited number of API calls per second, per API action (SendMessage, ReceiveMessage, or DeleteMessage).

It's not clear to me if `ChangeMessageVisibility` also supports "a nearly unlimited number of API calls per second". What do you think, should we risk it?

Reply:
I'm happy to start with 60/30 and then dial it down later.